content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-cli.md

```diff
@@ -0,0 +1,199 @@
+
+To review sample policies that you may want to leverage, please check out our page on [Chainguard Enforce Policy Examples](chainguard-enforce-policy-examples). If you would like to use the [Chainguard Enforce UI](https://console.enforce.dev) to work with policies, check out our guide on [How to create policies in the Chainguard Enforce Console](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-ui/).
+
+
+## Authenticate to the Chainguard Enforce platform
+
+To begin, you will need to authenticate to the Chainguard Enforce platform from your local machine with the following command.
+
+```sh
+chainctl auth login
+```
+
+A web browser window will open prompting you to log in using the [OIDC protocol](https://edu.chainguard.dev/software-security/glossary/#oidc). Once authenticated, you can begin using `chainctl` to manage and delegate your Chainguard Enforce policies.
+
+
+## Apply a new policy
+
+You can apply a policy to every cluster managed by a given group with the `policies apply` subcommand. Assuming that you have a policy file named `enforce-policy.yaml` in your current working directory, you could apply a policy with the following command.
+
+```sh
+chainctl policies apply -f enforce-policy.yaml
+```
+
+After running a `policies apply` command like this, an interactive menu will appear in your terminal prompting you to select the group that the policy should apply to. Chainguard Enforce provides a rich Identity and Access Management (IAM) model similar to those used by AWS and GCP. Each Chainguard policy needs to be associated with a group, and will be effective for that group as well as all the groups descending from it.
+
+The `-f` option in this command tells `chainctl` the location of the policy file to use to apply a new policy to a cluster. Note that all Chainguard Enforce policies must be written in YAML. If you don't include this option followed by a policy file, `chainctl` will open your default text editor for you to enter a policy manually.
+
+The `policies apply` subcommand accepts the `--group` option. When followed by the name or identification number of an existing Chainguard Enforce IAM group and combined with the `-f` flag and policy file, this option allows you to avoid using the interactive menu. This is useful in situations where you would prefer to take a more declarative approach to manage your policies.
+
+```sh
+chainctl policies apply -f enforce-policy.yaml --group my-group
```

Comment:
Can you specify in the text above that `my-group` is the example group and to replace with the relevant group name? Can you also use an ID for the group?